### PR TITLE
Random Variable rearrange

### PR DESCRIPTION
**Pull request by canyon289**
Rearranging distributions in alphabetical order (roughly right in draft state) to make it easier to navigate. Wont be offended if this we close this PR without merging if its not useful.

There should not be any changes to the code other than rearrangement

Here are a few important guidelines and requirements to check before your PR can be merged:
+ [X] There is an informative high-level description of the changes.
+ [X] The description and/or commit message(s) references the relevant GitHub issue(s).
+ [X] [`pre-commit`](https://pre-commit.com/#installation) is installed and [set up](https://pre-commit.com/#3-install-the-git-hook-scripts).
+ [X] The commit messages follow [these guidelines](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
+ [X] The commits correspond to [_relevant logical changes_](https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes), and there are **no commits that fix changes introduced by other commits in the same branch/BR**.  If your commit description starts with "Fix...", then you're probably making this mistake.
+ [X] There are tests covering the changes introduced in the PR.

Don't worry, your PR doesn't need to be in perfect order to submit it.  As development progresses and/or reviewers request changes, you can always [rewrite the history](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#_rewriting_history) of your feature/PR branches.

If your PR is an ongoing effort and you would like to involve us in the process, simply make it a [draft PR](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests).

### CODE REVIEW

**Comment by canyon289** [comment]
Changed to ready for review to trigger tests? Do tests trigger on draft prs?

File: aesara/_version.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 import sys
 from typing import Dict
 
+
```
Comment (brandonwillard) [inline aesara/_version.py:19]: This change isn't necessary.

**Review by brandonwillard — CHANGES_REQUESTED** [review]

File: aesara/_version.py (hunk 1 above)
Comment (canyon289) [reply aesara/_version.py:19]: Black automatically applied it. Ill remove it

**Comment by brandonwillard** [comment]
> Rearranging distributions in alphabetical order (roughly right in draft state) to make it easier to navigate. Wont be offended if this we close this PR without merging if its not useful.

It's going to take a little time to review this and make sure that something not directly tested (e.g. a name or special print form) didn't get changed or lost during the shuffle; otherwise, I'm not against reordering the classes.  

However, going forward, we're not going to enforce an ordering on these classes.  If we want something like that, we'll need to automate it, because it's not useful enough to warrant the ongoing manual effort.

**Comment by canyon289** [comment]
> > Rearranging distributions in alphabetical order (roughly right in draft state) to make it easier to navigate. Wont be offended if this we close this PR without merging if its not useful.
> 
> It's going to take a little time to review this and make sure that something not directly tested (e.g. a name or special print form) didn't get changed or lost during the shuffle; otherwise, I'm not against reordering the classes.
> 
> However, going forward, we're not going to enforce an ordering on these classes. If we want something like that, we'll need to automate it, because it's not useful enough to warrant the ongoing manual effort.

Dont worry about reviewing it immediately I know youre doing a lot already, and agree its not worth enforcing. One time should be good enough probably

**Comment by ricardoV94** [comment]
I added a bunch of RVs recently. Now might be a good time to rebase, rearrange and merge.

**Comment by canyon289** [comment]
Thanks Ricardo! Just gonna be honest on the list of things I have to do this one falls pretty low, particularly as its not that impactful in the scope of things. I can close this now and come back to it later, or we can put it on hold, or if someone else wants to take it over thatd be fine.

Current estimate I have if I finish this is probably mid summer at this point

**Comment by michaelosthege** [comment]
Looks like this PR is going stale and I would say the time needed for the rebase and also the reviews would be better spent on other things..
So I'm closing it. But anybody who wants to do the rebase just do it and reopen.

**Comment by canyon289** [comment]
Thanks @michaelosthege! I'll probably get back to it in a couple of months after GSOD, GSOC etc. 

Appreciate you taking action

**Pull request closed without merging**